idea/testData/intentions/convertClassToSealedClass/abstract.kt.after

```diff
@@ -0,0 +1,3 @@
+// INTENTION_TEXT: "Convert to sealed class"
```

Comment:
Since you aren't changing the intention text dynamically, there's no need to include INTENTION_TEXT in testdata files.